Marlin/src/module/tool_change.cpp

```diff
@@ -1068,7 +1068,8 @@ void tool_change(const uint8_t new_tool, bool no_move/*=false*/) {
     #endif
 
     #ifdef EVENT_GCODE_AFTER_TOOLCHANGE
-      gcode.process_subcommands_now_P(EVENT_GCODE_AFTER_TOOLCHANGE);
+      if (!no_move)
+        gcode.process_subcommands_now_P(PSTR(EVENT_GCODE_AFTER_TOOLCHANGE));
```

Comment:
Looks like `no_move` has a bigger meaning now. Probably time to rename it to something like "`is_internal`."

Reply:
Still just blocking movement. I think the current name is clearer on that intent.